modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/ChallengeTag.java

```diff
@@ -36,11 +36,11 @@ public void setMeta(MetaTag meta) {
         this.meta = meta;
     }
 
-    public void buildChallElement(Document doc, Element rootElement) {
-        Element chal = doc.createElement(Constants.CHALLENGE);
-        rootElement.appendChild(chal);
+    public void buildChalElement(Document doc, Element rootElement) {
```

Comment:
Please refactor the method name to something like buildChallengeElement which looks more meaningful.